rootfs/api/models/build.py

```diff
@@ -52,8 +52,8 @@ def version(self):
         return 'git-{}'.format(self.sha) if self.source_based else 'latest'
 
     def create(self, user, *args, **kwargs):
-        latest_release = self.app.release_set.latest()
-
+        latest_release = self.app.release_set.filter(failed=False).latest()
```

Comment:
since the deploy v4 failed wouldn't the user do the same deploy again because it failed.